templates/partials/answers/checkbox.html

```diff
@@ -1,12 +1,13 @@
 {% from "components/checkboxes/_macro.njk" import onsCheckboxes %}
 
 {{ onsCheckboxes({
-  "id": answer["id"],
   "legend": question_title,
+  "id": answer.id,
   "legendClasses": "u-vh",
   "checkboxesLabel": '' if answer.label == '' else answer.label or _("Select all that apply"),
   "checkboxes": map_checkbox_config(form, answer),
   "mutuallyExclusive": mutuallyExclusive,
   "error": error
+  }
```

Comment:
Got an extra bracket here that is causing an error
```suggestion
```

Reply:
OK thanks!